demos/python_demos/whiteboard_inpainting_demo/utils/network_wrappers.py

```diff
@@ -0,0 +1,123 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import json
+import logging as log
+
+import cv2
+import numpy as np
+
+from utils.ie_tools import IEModel
+from .segm_postprocess import postprocess
+
+
+class MaskRCNN(IEModel):
+    def __init__(self, ie, model_path, conf=.6, device='CPU', ext_path='', max_reqs=1):
+        super().__init__(ie, model_path, conf, device, ext_path, max_reqs)
+
+        required_input_keys = [{'im_info', 'im_data'}, {'im_data', 'im_info'}]
+        current_input_keys = set(self.inputs_info.keys())
+        assert current_input_keys in required_input_keys
+        required_output_keys = {'boxes', 'scores', 'classes', 'raw_masks'}
+        assert required_output_keys.issubset(self.net.outputs)
+
+        self.n, self.c, self.h, self.w = self.inputs_info['im_data'].shape
+        assert self.n == 1, 'Only batch 1 is supported.'
+
+    def get_allowed_inputs_len(self):
+        return (2, )
+
+    def get_allowed_outputs_len(self):
+        return (4, 5)
+
+    def _preprocess(self, frame):
+        image_height, image_width = frame.shape[:2]
+        scale = min(self.h / image_height, self.w / image_width)
+        processed_image = cv2.resize(frame, None, fx=scale, fy=scale)
+        processed_image = processed_image.astype('float32').transpose(2, 0, 1)
+
+        sample = dict(original_image=frame,
+                      meta=dict(original_size=frame.shape[:2],
+                                processed_size=processed_image.shape[1:3]),
+                      im_data=processed_image,
+                      im_info=np.array([processed_image.shape[1], processed_image.shape[2], 1.0], dtype='float32'))
+        return sample
```

Comment:
I suggest returning a tuple instead of dict and unpack it later. Or if you insist on giving names to each of element, you should use namedtuple.
You don't need to give a name to an object you return. Just return whatever you construct.

Reply:
Fixed with tuple.